bridge/engine/EngineBarbell.h

```diff
@@ -68,6 +68,8 @@ struct EngineBarbell
     bridge::RtpMap videoRtpMap;
     bridge::RtpMap videoFeedbackRtpMap;
 
+    uint32_t minEstimatedUplinkBitrate;
```

Comment:
The name could mean a variety of things right? We know atm, but perhaps it should be made clear it is not about barbell bandwidth and it is not a bitrate. 
eg.
minClientDownlinkBandwidth

Reply:
Fixed.